### DISCUSSION TITLE
Table column natural sort

### DISCUSSION THREAD

**Discussion by etfz** [grafana/grafana#65030] [Dashboards, Panels & Visualizations]
Hi,

Similar to https://github.com/grafana/grafana/issues/13179, which I believe is for something else, I would like to have natural sorting for table columns. I have tables of switch ports (eg `ether[1-48]`) which are not sorted in a helpful manner.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82752. Feel free to continue the discussion around this there. Thank you!
